docs/MESSAGE_TRACING_IDENTIFIERS.md

```
@@ -34,7 +34,7 @@ var correlationId = ActivityExtensions.GetCorrelationId();
 
 ---
 
-### 2. CausationId (Guid)
+### 2. CausationId (Guid or string)
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)